flow-nodes/api-builder-plugin-petstore/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('@axway/api-builder-oas-flow-node')(module);
```

Comment:
I had difficulty generating credentials - the issue was I installed the example from a directory, which links the module.  It highlights a potential problem with this kind of example.  Each plugin is calculating their own `isDeveloperMode`, I think this should be done in runtime and passed in as config.

Reply:
Maybe. i actually missed the actual change to OAS when we put in that isdeveloperMode stuff. I don't really like the log that gets printed out since there's no such thing as "developer mode" anywhere, it's a new concept to users that isn't documented anywhere so they'll have no idea what it is. We don't even call it developer mode when we talk about it :p 

maybe users are probably more likely to understand if we say something like it looks like it's running a production install instead or at least exactly why we're not generating credentials without any mention of modes so there's no confusion.

But I digress, yeah, it can be something extra from config, although we'd still have to try and require admin for backwards compatibility.  That said, if we can't require the correct admin or runtime from the plugin when it's linked up, then we won't have access to the credential store either no matter what we got passed in as options right?

Reply:
Agreed - there is nothing to do for this ticket, the `isDeveloperMode` and the additional peer-dependency use of runtime _within_ the plugin is problematic.  We need a clear/consistent way of plugging credentials.